src/qml/qgismobileapp.qml

```diff
@@ -699,6 +736,48 @@ ApplicationWindow {
         displayToast( qgisProject.topologicalEditing ? qsTr( "Topological editing turned on" ) : qsTr( "Topological editing turned off" ) );
       }
     }
+
+    QfToolButton {
+      id: freehandButton
+      round: true
+      visible: stateMachine.state === "digitize"
```

Comment:
When one is working on a device without a pen, there still will be this button the user neither uses nor knows why it's here. Correct?

Is it controllable over `PointerDevice.Stylus`? Or should it be a setting to enable freehand digitizing?

Reply:
@signedav , good point. I'll make the button only show if a mouse or stylus is hovering the canvas.